include/Util/helper.php

```diff
@@ -143,7 +143,11 @@ function join_array_elements_with_delimiter( $value, $delimiter = ',' ) {
  * @return string
  */
 function get_user_defined_date_time_format() {
-	return sprintf( '%1$s %2$s', get_option( 'date_format', 'Y-m-d' ), get_option( 'time_format', 'g:i a' ) );
+	return sprintf(
```

Comment:
The 3 scrutinizer issues that I reported all belong to the `get_masked_value` function. I was not expecting you to fix this function :)